api/envoy/api/v2/core/health_check.proto

```diff
@@ -44,6 +44,10 @@ message HealthCheck {
     (gogoproto.stdduration) = true
   ];
 
+  // An optional jitter amount in milliseconds. If specified, Envoy will start health
+  // checking after initial_jitter.
```

Comment:
Can you clarify that this is a random value between 0 and initial_jitter? (Or something like that)

Reply:
ok